apps/web/app/[locale]/timesheet/[memberId]/components/CalendarView.tsx

```
@@ -32,31 +33,39 @@ export function CalendarView({ data, loading, user }: { data?: GroupedTimesheet[
-                    <div className="flex items-center justify-center h-full min-h-[280px]">
-                        <p>{t('pages.timesheet.NO_ENTRIES_FOUND')}</p>
-                    </div>
-                )
-            ) : (
-                <div className="flex items-center justify-center h-full">
-                    <p>{t('pages.timesheet.LOADING')}</p>
-                </div>
-            )}
+                    <CalendarDataView data={data} t={t} />
+                )}
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)